tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -1655,6 +1653,30 @@ public function updateSpaceQuota(
 				$spaceId
 			)
 		);
+		$res = $this->featureContext->getJsonDecodedResponse();
+	}
+
+	/**
+	 * @Given /^user "([^"]*)" has changed the quota of the "([^"]*)" space to "([^"]*)"$/
```

Comment:
maybe?
```suggestion
	 * @Given /^user "([^"]*)" has changed the quota of the personal space of "([^"]*)" to "([^"]*)"$/
```

Reply:
If it's possible to change the quota of shares jail this step could also be used to change the quota of it as well
```
Given "Brian" has changed the quota of  "Shares Jail" space to "<something>"
```
But I guess we can create a separate step for that too? What do you recommend?

Reply:
https://github.com/owncloud/ocis/issues/4472#issuecomment-1230226840 according to this it won't be possible.